next.clayui.com/content/docs/components/select.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: "Select"
+description: "A form control element used to select from several provided options and enter data."
+packageNpm: "@clayui/select"
+packageVersion: "3.0.0"
+packageStatus: "Beta"
+---
+
+import {Select, SelectWithOption} from '../../../src/components/clay/Select';
+
+<div class="nav-toc">
+
+- [Variants and Best Pratices](#variants-and-best-pratices)
+    - [SelectWithOption](#selectwithoption)
+- [API](#api)
+
+</div>
+
+<div class="clay-site-alert alert alert-info">
+	Check the <a href="https://liferay.design/lexicon/">Lexicon</a> <a href="https://liferay.design/lexicon/core-components/forms/selector/">Selectors Pattern</a> for a more in-depth look at the motivations and proper usage of this component.
+</div>
+
+## Variants and Best Pratices
+
+### Default
+
+This pattern is completely different from a dropdown. A selector is a native component with a data provider. A dropdown, however, opens a panel that could contain multiple and different natural elements.
```

Comment:
I'm not sure the references to dropdown are helpful here. A user may have no idea or concept of clay's dropdown. Rather I think we can reference the native component portion and link to both clay's styling of the native <select> element.

Reply:
Solved!